File: TP1/c++/Pixel.cpp
#include "Pixel.hpp"

Pixel::Pixel() : r(0), g(0), b(0) {

}

Pixel::~Pixel() {

}

void Pixel::print() {
	std::cout << this->r << " ";
}

int Pixel::getR() {
	return this->r;
}

int Pixel::getG() {
	return this->g;
}

int Pixel::getB() {
	return this->b;
}

void Pixel::set(int r, int g, int b) {
	this->r = r;
	this->g = g;
	this->b = b;
}

void Pixel::setR(int n) {
	this->r = n;
}

void Pixel::setG(int n) {
	this->g = n;
}

void Pixel::setB(int n) {
	this->b = n;
}

File: TP1/java/Pixel.java
public class Pixel {
    private int r, g, b;

    public Pixel() {
        this.r = 0;
        this.g = 0;
        this.b = 0;
        // System.out.println("classe pixel declaree!");
    }

    public int getR() {
        return this.r;
    }

    public int getG() {
        return this.g;
    }

    public int getB() {
        return this.b;
    }

    public void set(int r, int g, int b) {
        this.r = r;
        this.g = g;
        this.b = b;
    }
}